README.md

```diff
@@ -51,23 +53,24 @@ Implement the logic for your module by updating the `main.tf`, `version.tf`, `va
 
 ### Create examples and tests
 
-Add one or more examples in the `examples` directory that consume your new module, and configure tests for them in the `tests` directory.
+Add one or more examples in the `examples` directory that consume your new module, and configure tests for them in the `tests` directory. For more information about tests, see [Tests](https://terraform-ibm-modules.github.io/documentation/#/tests).
 
 ### Update the content in the readme file
 
 After you implement the logic for your module and create examples and tests, update this readme file in your repository by following these steps:
 
 1.  Update the title heading and add a description about your module.
-1.  Update the badge links.
-1.  Remove all the content in this H2 heading section.
-1.  Complete the [Usage](#usage), [Required IAM access policies](#required-iam-access-policies), and [Examples](#examples) sections. The [Requirements](#requirements) section is populated by a pre-commit hook.
+2.  Update the badge links.
```

Comment:
Is the numbering not preserved even when the list is not indented? It's easier to maintain if every list item starts with `1. `.